### PR TITLE
Exponential histogram blog

### LINKED ISSUE #1639: Repost exponential histogram blog post
I recently wrote / published a blog post about otel exponential histograms on [New Relic's blog](https://newrelic.com/blog/best-practices/opentelemetry-histograms).

Its only got 1 or 2 vendor specific references that are easy to remove, after which its just a good run down of an awesome new open telemetry feature. I think it would be a good addition to the opentelemetry blog, but before I go through the process of converting it to markdown and making it completely vendor agnostic, I want to confirm that reposting existing content is ok.

Give me a 👍 and I'll get PR out there shortly.

### PR DESCRIPTION
**Pull request by jack-berg**
Resolves #1639.

### CODE REVIEW

File: layouts/partials/hooks/head-end.html
Comment (chalin) [inline layouts/partials/hooks/head-end.html:4]: Thanks for this @jack-berg, but I'd rather that we address this via a separate PR (possibly with this approach or another).

File: content/en/blog/2022/exponential-histograms.md
Comment (chalin) [inline content/en/blog/2022/exponential-histograms.md:3]: How about the following as a short title?

> Exponential Histograms

File: content/en/blog/2022/exponential-histograms.md
Comment (chalin) [inline content/en/blog/2022/exponential-histograms.md:37]: Here and elsewhere in the file, link to the local spec pages under [/docs/reference/specification/](https://opentelemetry.io/docs/reference/specification/). For example, the link on this line would become `/docs/reference/specification/metrics/api/`.

File: content/en/blog/2022/exponential-histograms.md
Comment (chalin) [inline content/en/blog/2022/exponential-histograms.md:110]: Please create a page bundle for this post. That is:

- Create a directory named `content/en/blog/2022/exponential-histograms`
- Rename `content/en/blog/2022/exponential-histograms.md` to `content/en/blog/2022/exponential-histograms/index.md`
- Move all images to be under under `content/en/blog/2022/exponential-histograms` and adjust the image paths to be local.

Let me know if you feel like you need more guidance or would like for me to help.

**Review by chalin — COMMENTED** [review]
See inline comments, thanks.

File: layouts/partials/hooks/head-end.html
Comment (jack-berg) [reply layouts/partials/hooks/head-end.html]: Seems reasonable, split out to #1649.

File: content/en/blog/2022/exponential-histograms/index.md
Comment (svrnm) [inline content/en/blog/2022/exponential-histograms/index.md:228]: Ideally the image title gives a description on what you can see on that picture.

**Review by svrnm — CHANGES_REQUESTED** [review]
thanks, that's some awesome content:)

A few suggested changes.

File: content/en/blog/2022/exponential-histograms/index.md
Comment (jack-berg) [reply content/en/blog/2022/exponential-histograms/index.md:228]: I've tried to give all the images filenames that reflect what they represent. The image titles are therefore just the filenames. I can remove the dashes?

File: content/en/blog/2022/exponential-histograms/index.md
Comment (chalin) [inline content/en/blog/2022/exponential-histograms/index.md:162]: Here and elsewhere, my _preference_ would be for the "math" to be at least in plain text, rather than code font. If you do want to style it, then (math) variables should be in italics. See the suggestion below. 

But I understand that it might not be worth it at this point, so I leave it up to you @jack-berg.

In particular, why is the (plain) number two in code font?

```suggestion
buckets. Each bucket count is an 8-byte value, so an _N_ bucket exponential
bucket histogram is encoded as (_N_+2) 8-byte values (_N_ bucket counts and 2
```

Which looks like this:

> image

File: content/en/blog/2022/exponential-histograms/index.md
Comment (chalin) [inline content/en/blog/2022/exponential-histograms/index.md:277]:
```suggestion
To enable exponential bucket histograms in other languages, see the language-specific documentation under [Instrumentation][].
```

If you agree with this change then you'll need to change the link definition on line 307 below from

    [opentelemetry.io]: /docs/instrumentation

to

    [Instrumentation]: /docs/instrumentation

File: content/en/blog/2022/exponential-histograms/index.md
Comment (chalin) [inline content/en/blog/2022/exponential-histograms/index.md:13]:
```suggestion
[percentiles](https://en.wikipedia.org/wiki/Percentile).
```

The parenthetical remark doesn't add much. Also, AFAICT, "p" followed by a number is standard notation for a percentile.

File: content/en/blog/2022/exponential-histograms/index.md
Comment (chalin) [inline content/en/blog/2022/exponential-histograms/index.md:4]:
```suggestion
date: 2022-08-23
```

**Review by chalin — COMMENTED** [review]
A few tweaks and one optional suggested format change. For details, see inline comments.

File: content/en/blog/2022/exponential-histograms/index.md
Comment (svrnm) [reply content/en/blog/2022/exponential-histograms/index.md:162]: @chalin I am wondering, if hugo allows some setting of math code like github markdown allows it these days. I think that especially the syntax `base**i` might not be understood by everyone, whereas $base^i$ is a notation that people should know about. 

If not, it would be good to introduce them verbally:

> and has bucket boundaries `(base**i, base**(i+1)]`, where `base**i` means that base is raised to the power of i.

Comment (chalin) [reply content/en/blog/2022/exponential-histograms/index.md:162]: We can enable support for math via Docsy. @jack-berg - let us know how you'd like to deal with this.

File: content/en/blog/2022/exponential-histograms/index.md
Comment (jack-berg) [reply content/en/blog/2022/exponential-histograms/index.md]: This was resolved when you ran prettifier, I think.

File: content/en/blog/2022/exponential-histograms/index.md
Comment (jack-berg) [reply content/en/blog/2022/exponential-histograms/index.md:162]: I think italics is fine, updating. I'd rather just explain that `**` means "raised to the power of" then fuss around trying to get math support.

File: content/en/blog/2022/exponential-histograms/index.md
Comment (svrnm) [inline content/en/blog/2022/exponential-histograms/index.md:4]: I think we should update this to 24: having multiple blog posts with the same date, leads to some strange behaviour, see https://github.com/open-telemetry/opentelemetry.io/pull/1659

Comment (chalin) [reply content/en/blog/2022/exponential-histograms/index.md:4]: Agreed, let's make this today's date.

Other than that, is the ready for your approval @svrnm?
If so, after you approve, I'll update the date and then merge.

Comment (jack-berg) [reply content/en/blog/2022/exponential-histograms/index.md]: Ha, forgot what day it was. Fixed.

**Comment by jack-berg** [comment]
Ready to merge this?

**Review by chalin — APPROVED** [review]
LGTM. Given @svrnm's last comment of "thanks, that's some awesome content:)" ... and that you've addressed the issues he raised, I'm going to merge.

@svrnm - if you think that there's still some work to be done, we can always address it in a followup PR.

Thanks all!

**Pull request merged**
